stdlib/2and3/zipfile.pyi

```diff
@@ -133,13 +130,13 @@ class ZipInfo:
     CRC: int
     compress_size: int
     file_size: int
-    def __init__(self, filename: Optional[Text] = ...,
-                 date_time: Optional[_DT] = ...) -> None: ...
+    def __init__(self, filename: Optional[Text] = ..., date_time: Optional[_DT] = ...) -> None: ...
     if sys.version_info >= (3, 6):
         def is_dir(self) -> bool: ...
+        @classmethod
+        def from_file(cls, filename: _Path, arcname: Optional[_Path] = None) -> ZipInfo: ...
```

Comment:
`arcname` does not accept a `_Path`, only a `str` as far as I can see. Also, the default value should be `...` (this is why the build fails).

Reply:
Pushing a change for the default argument.
I think `arcname` does accept `_Path`, since only `os.path.splitdrive(arcname)` is the only function called on `arcname` and `os.path.splitdrive` does accept `builtins._PathLike[str]` in python >= 3.6.  I can also get `z = ZipInfo.from_file(Path('./file'), Path('./file))` to work or at least not error in python 3.6.9. 
Thanks for the quick review!